common/src/main/java/com/moparisthebest/jdbc/codegen/JdbcMapper.java

```diff
@@ -48,6 +48,13 @@
 		DatabaseType databaseType() default DatabaseType.DEFAULT;
 		String arrayNumberTypeName() default "";
 		String arrayStringTypeName() default "";
+
+		/**
+		 * Allows consumer of JdbcMapper.Mapper to generate DaoBean with Spring Boot Stereotype @Repository
+		 * Defaults to false
+		 * @return
+		 */
+		OptionalBool generateAsSpringBean() default OptionalBool.FALSE;
```

Comment:
The default should be OptionalBool.DEFAULT, so if it's not set at the class level, it can be set as an annotation processor option.  Maybe the hard-coded default if neither of these are set could be true if org.springframework.stereotype.Repository is on the compile-time classpath?